init.lua

```diff
@@ -27,13 +36,39 @@ qa_block.smartfs = smartfs
 dofile(qa_block.modpath.."/smartfs_forms.lua")
 local smartfs_enabled = true
 
+
+-----------------------------------------------
+-- Return a list of keys sorted - Useful when looking for regressions
+-- https://www.lua.org/pil/19.3.html
+-- Additional helper to be used in checks
```

Comment:
Due to table insertions and sorting, this performs way worse than `pairs`. `minetest.registered_items` can contain tens of thousands entries, hence using this function there is wasteful and unnecessary. Code should never expect a certain order in `pairs`, thus existing code that relies on it is outright wrong and needs fixing instead of relying on such a function.